pkgs/development/python-modules/python-dbusmock/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib, buildPythonPackage, fetchFromGitHub, runtimeShell,
   nose, dbus, dbus-python, pygobject3,
-  which, pyflakes, pycodestyle, bluez, networkmanager
+  which, pyflakes, pycodestyle, bluez, networkmanager,
+  doCheck ? true
```

Comment:
I am not really fond of adding more control flags as parameters. I would prefer relying on https://github.com/NixOS/nixpkgs/pull/119942, if it works with `buildPythonPackage`.